src/overlays/actors/ovl_En_Insect/z_en_insect.c

```diff
@@ -74,7 +83,15 @@ static ColliderJntSphInit sColliderInit = {
     sColliderItemInit,
 };
 
-static u16 D_80A7DF10[] = { 0, 5, 7, 7 };
+/**
+ * The initial flags for a new bug, depending on its type.
+ */
+static u16 sInitInsectFlags[] = {
+    0,
+    INSECT_FLAG_0 | INSECT_FLAG_IS_SHORT_LIVED,
+    INSECT_FLAG_0 | INSECT_FLAG_1 | INSECT_FLAG_IS_SHORT_LIVED,
+    INSECT_FLAG_0 | INSECT_FLAG_1 | INSECT_FLAG_IS_SHORT_LIVED,
+};
```

Comment:
can prob afford to just be a // comment, its taking up alot of space and looks like a function comment at a glance